fiducial_follow/nodes/follow.py

```diff
@@ -58,20 +69,74 @@ def __init__(self):
        # This allows to visualize the 3D position of the fiducial easily in rviz
        self.br = tf2_ros.TransformBroadcaster()
 
-       # A publisher for robot motion commands
-       self.cmdPub = rospy.Publisher("/cmd_vel", Twist, queue_size=1)
 
        # Flag to avoid sending repeated zero speeds
        self.suppressCmd = False
 
-       # The name of the coordinate frame of the fiducial we are interested in
+       # define a move time that when non-zero we continue to do rotate or drive commands
+       self.moveTimeLeft = 0.0
+
+       # ---------------------- start of keywords for use in commands ---------------------------
+
+       # define a fiducial that when set implies we are not seeking any fiducial
+       self.null_fiducial = "fid0"
```

Comment:
This seems a bit hacky, could we just set `target_fiducial` to None?

Reply:
Yes and in fact the readme says exactly that.   I will change this to say 'none' to be more clear.  It ends up being a string compare anyway so 'none' is ok.

Reply:
I meant the keyword None not the string none, but that would work

Reply:
I meant the keyword None not the string none, but that would work

Reply:
I meant the keyword None not the string none, but that would work